### ISSUE TITLE
Add the culture to requests/responses in public facing cloudscribe pages

### ISSUE THREAD

**Issue by LaineS** [cloudscribe/cloudscribe#998]
![image](https://github.com/cloudscribe/cloudscribe/assets/6085799/9d87fecf-bc09-4995-ad47-226522db609b)

This ensures the culture is persisted the the Login and user Registration process. Key pages include:

![image](https://github.com/cloudscribe/cloudscribe/assets/6085799/966dd0fe-c925-498d-a45e-6038266110e2)

**NB** Also need to ensure StringResources are used in all pages (there are a few hard-coded strings in these pages. Also, check the controllers as they send back hard-coded error messages direct to the UI. Could use an error code and the a StringResources in the cshtml)

**Comment by markidox** [comment]
I think the first part is actually for #935  I must have added it in here by mistake. I'm pretty sure I did test it in a multi-tenant environment and it worked fine, which was surprising as I couldn't see why we'd create the RedirectToSiteRoot method in that case...

**Comment by JimKerslake** [comment]
Yeah I'm not sure that routing explicitly to ("Index", "Home") is always going to be the same thing as just linking to /siteFolder/   
since it's only by our routing config we specify that the default route for  /   maps to Home/Index
and in some circumstances / could potentially be configured to map to a different controller/action, in which case this would be a bug.

**Comment by CrispinF** [comment]
@markidox @JimKerslake please clarify... getting on with what exactly? I'm not clear what we are trying to achieve on this ticket, and where it's ended up?

**Comment by LaineS** [comment]
In Claude Code, try this (change XXXX and point it to a test folder, then if it works well expand to cover more folders):

We want to replace all text strings in our CSHTML files with a StringReference in a resx file. 

1. We use the StringReference library in our code as follows sr["my string here"]
2. In many cases, we do not have the corresponding entry in the resx file here cloudscribe.templates\Content\WebApp\GlobalResources\
3. Search through the cshtml files in the following folder XXXXX. Replace the sr["my string reference"] with a new value sr["new_value"] and then add this to the resx file also
4. The new value you chose should be a short but descriptive set of words (with underscors) representing the existing str
5. Also replace any free text you see in these files too

**Comment by JimKerslake** [comment]
@LaineS  - this prompt has worked well for me... I cheekily asked it to translate into Welsh also, so we'll see how good that has been..!

We want to replace all text strings in our CSHTML files with a StringReference in a resx file.

1 - We use the StringReference library in our code with the format as follows:  sr["my string here"]

2 - Sometimes we also use reference format as follows:  StringLocalizer["my string here"];

3 - In all of these cases, sr  or  StringLocalizer are declared at the top of the code as  IStringLocalizer<CloudscribeCore> 

4 - Ignore any cases where sr or  StringLocalizer are declared at the top of the code as  IStringLocalizer<QueryToolResources> 

5 - In many cases, we do not have the corresponding entry in the resx file here:  sourceDev.WebApp\GlobalResources\CloudscribeCore.en-US.resx

6 - Search through the cshtml files in the following folder:  cloudscribe.Core.CompiledViews.Bootstrap5\Views\Account   Replace the sr["my string reference"] with a new value sr["new_value"] and then add this to both the CloudscribeCore.en-US.resx file (if it does not exist in that file already) and the  CloudscribeCore.en-GB.resx file (if it does not exist in that file already).

7 - The new value you choose should be a short but descriptive set of words (with underscores) representing the existing string

8 - The new value you insert should also have a comment added to it:  "verified present - Claude" 

9 - Also replace any free text you see in these files that is not already stringreferenced in that manner (ignoring comments), by creating an appropriate stringreference for it, and inserting that into the two ResX files, with the same comment, in the pattern above.

10 - For each new value you insert, also insert the same key in CloudscribeCore.cy.resx, but its Value should be the Welsh translation of the string used in the US and GB files.
